datastore/__init__.py

```diff
@@ -81,10 +81,14 @@ async def new_user(cursor, email, connection_string, tier="user") -> User:
         blob_service_client = BlobServiceClient.from_connection_string(
             connection_string
         )
-        container_client = blob_service_client.create_container(f"{tier}-{user_uuid}")
+        container_client = blob_service_client.create_container(
+            azure_storage.build_container_name(str(user_uuid), "user")
```

Comment:
```
Defined in datastore/db/queries/user/__init__.py
def register_user(cursor, email: str) -> None:
```

```Line 79 user_uuid = user.register_user(cursor, email)```

Currently register_user is defined to return None, can we update that to str?

Reply:
made it a UUID instead

Reply:
This was mentionned as an issue in #201